cluster/gce/gci/configure-helper.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+qq  #!/bin/bash
```

Comment:
Is that a typo? I have no idea what it's supposed to do and it breaks the tests.